src/main/java/com/mycompany/app/App.java

```diff
@@ -16,6 +16,8 @@
 
 package com.mycompany.app;
 
+import com.esri.arcgisruntime.ArcGISRuntimeEnvironment;
+import com.esri.arcgisruntime.mapping.BasemapStyle;
```

Comment:
these are fine but there's an unused import below now we're using BasemapStyle

import com.esri.arcgisruntime.mapping.Basemap;